view/theme/frio/js/theme.js

```diff
@@ -300,6 +300,14 @@ $(document).ready(function(){
 		});
 	});
 
+	// Customize some elements when the app is used in standalone mode on Android
+	if (window.matchMedia('(display-mode: standalone)').matches) {
+		// Open links to source outside of the webview
+		$('.plink').on('click', function (e) {
```

Comment:
Wouldn't it be better to have something like `$('body').on('click', '.plink', function (e) {` ? I guess otherwise it wouldn't work with infinite scrolling